recipes/cppserver/all/patches/00001-cmake-clean-up.patch

```diff
@@ -0,0 +1,141 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 5a21eed9..94135f07 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -18,50 +18,50 @@ if(DOXYGEN_FOUND)
+ endif()
+ 
+ # CMake module path
+-set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
```

Comment:
My only comment about this patch is that it is hard to maintain for future cppserver releases.

Ideal is to create a patch adding options that can be set in the conanfile.
A patch could then be sent upstream.

Reply:
Yes, I agree. The other projects of the same author that are dependencies of this one suffer from the same issue.

Maybe we can go on with this PR the way it is for now, and I commit myself to send PRs upstream to handle this more smoothly.

What do you think?

Reply:
> Maybe we can go on with this PR the way it is for now, and I commit myself to send PRs upstream to handle this more smoothly.

Sounds good

Reply:
> What do you think?

:notes:  to my :ear: